### PR TITLE
testing: rollout 35.20211029.2.0

### PR DESCRIPTION
**Pull request by gursewak1997**
```
testing
    version: 35.20211029.2.0 (latest)
    start: 2021-11-10 14:00:00+00:00 UTC (in 16:47:58)
           2021-11-10 09:00:00-05:00 Raleigh/New York/Toronto
           2021-11-10 15:00:00+01:00 Berlin/France/Poland
    duration: 4320m (72.0h)

```

### CODE REVIEW

File: streams/testing.json [hunk 1]
```diff
@@ -1,92 +1,92 @@
 {
     "stream": "testing",
     "metadata": {
-        "last-modified": "2021-11-02T15:06:03Z",
-        "generator": "fedora-coreos-stream-generator v0.1.0-7-gfe56f7b"
+        "last-modified": "2021-11-09T20:34:36Z",
+        "generator": "fedora-coreos-stream-generator v0.1.0-12-gd500e09"
```
Comment (bgilbert) [inline streams/testing.json:5]: This is equivalent to `v0.2.0`, but interestingly, your local repo didn't include the `v0.2.0` tag when you built the generator.

Comment (bgilbert) [reply streams/testing.json:5]: (Odd, but seems harmless.)

Comment (dustymabe) [reply streams/testing.json:5]: Is it possible he somehow pulled the latest code but not the tags?

**Review by bgilbert — DISMISSED** [review]
The update barrier was lost in the force-push.

File: streams/testing.json (hunk 1 above)
Comment (bgilbert) [reply streams/testing.json:5]: It's possible.  @gursewak1997, any chance you disabled fetching tags, or did a shallow clone?

Comment (gursewak1997) [reply streams/testing.json:5]: It could be. Will give a quick check and will re do it if it did happen.

**Review by dustymabe — APPROVED** [review]
LGTM

File: streams/testing.json (hunk 1 above)
Comment (gursewak1997) [reply streams/testing.json:5]: My fork's got only one tag so will update it.

**Review by bgilbert — DISMISSED** [review]

**Review by bgilbert — APPROVED** [review]

**Review by dustymabe — APPROVED** [review]

**Pull request merged**